### DISCUSSION TITLE
Can we have a release please?

### DISCUSSION THREAD

**Discussion by rajaramsrn** [ecederstrand/exchangelib#908] [General]
Hello,

Thanks a bunch for the commit https://github.com/ecederstrand/exchangelib/commit/57dffb37ef2b4c2d6f1353144faac730ec688baf . This specific issue is breaking several of my deployments.  Could we have a release please?

Raj

**Comment by ecederstrand** [comment]
I have just released version 4.3.0: https://pypi.org/project/exchangelib/
